ioctl/cmd/hdwallet/hdwalletcreate.go

```diff
@@ -68,18 +69,22 @@ func hdwalletCreate() error {
 	entropy, _ := bip39.NewEntropy(128)
 	mnemonic, _ := bip39.NewMnemonic(entropy)
 
-	out, err := util.EncryptString(mnemonic, password)
+	seed, _ := hdwallet.NewSeedFromMnemonic(mnemonic)
+
+	out, err := util.Encrypt(seed, []byte(password))
 	if err != nil {
 		return output.NewError(output.ValidationError, "failed to encrypting mnemonic", nil)
 	}
 
-	if err := ioutil.WriteFile(hdWalletConfigFile, []byte(out), 0600); err != nil {
+	out = append(out, util.HashSHA256(seed)...)
```

Comment:
this leaves last 32 bytes of the config file = hash of seed

should encrypt(  seed + HashSHA256(seed) ) to avoid this